src/Tabs.js

```diff
@@ -1,6 +1,6 @@
-import classNames from 'classnames';
```

Comment:
I don't think we want this because we'll be dropping support for horizontal tabs in the next release anyway – no reason to make any changes to deprecated code.